lib/main.dart

```diff
@@ -1,6 +1,8 @@
+import 'package:flutter/material.dart';
 import 'core/common/enums/environment.dart';
 import 'main_common.dart';
 
 void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
```

Comment:
what is this?

Reply:
we just ensure that all widget will created before run the application !